graphics/openvdb/Portfile

```diff
@@ -61,28 +70,30 @@ configure.args-append   -DUSE_CCACHE=OFF \
 configure.args-append   -DOPENVDB_BUILD_UNITTESTS=OFF
 
 set pythons_suffixes    {27 36 37 38 39}
-
 set pythons_ports       {}
-foreach s ${pythons_suffixes} {
+foreach s $pythons_suffixes {
     lappend pythons_ports python${s}
 }
 
-foreach s ${pythons_suffixes} {
+foreach s $pythons_suffixes {
     set p python${s}
     set v [string index ${s} 0].[string index ${s} 1]
-    set i [lsearch -exact ${pythons_ports} ${p}]
-    set c [lreplace ${pythons_ports} ${i} ${i}]
-    variant ${p} description "Build the Python ${v} bindings" conflicts {*}${c} "
+    set i [lsearch -exact $pythons_ports ${p}]
+    set c [lreplace $pythons_ports ${i} ${i}]
```

Comment:
Hey Jason, this is another nitpicky-esque comment. But again, it comes down to tangible changes, vs. those that are aren't:

From a consistency perspective, why change the variable referencing convention from `${varname}`, to `$varname`? The former is used everywhere else in this port. Was this just a simple oversight when committing these changes...?

Reply:
This is purely a stylistic preference on my part, but I prefer to keep variables that have been declared elsewhere, like in PortGroups or global variables from base, as `${varname}`, and variables that have been declared by me in "this" Portfile, as `$varname` (obviously the exception is when the brackets are required, e.g. `${varname}.fileext`).

It lets me see at a glance whether the variable is local or not. Since it looks like I'm going to be taking over as maintainer of the port, I thought that I would take the opportunity to format things in a way that makes it easier for me to read. Also, it cuts down on file size, although this benefit is rather negligible.

Reply:
Oh, and as far as why I _do_ leave the single-letter variables as `${varname}`, it's because I'm a bit ambivalent about those. I have been bitten before where I have left the variables as `$i`, but then ended up doing something like `$iotherstuff` (when it should have been `${i}otherstuff`) and confused myself. 😝 

I might end up deciding to get rid of the brackets around the single-letter variables in a future PR... like I said, I'm ambivalent at the moment, lol.

Reply:
Well, this is another maintainer's port. And while folks are free to follow their own personal preferences for those that they maintain, we should remain consistent with the existing conventions for someone else's port.

As an example: If you submitted a PR for one of my ports, and you chose to make unnecessary changes like this, I'd ask you to promptly remove them.

And ultimately there's no way I'd merge it otherwise. So you'd either have to make the changes requested, or the PR would be rejected outright.

Ditto for this one: This port is maintained by Marcus, and there's a consistent convention in place. And there's simply no way that I'll merge this in it's current state.

If someone else is willing to - or Marcus chooses to - that's fine. But I can't, in good conscious, merge this as-is.

Reply:
Updated the commit, to limit the number of unnecessary changes.